readthedocs/builds/storage.py

```diff
@@ -87,14 +88,6 @@ def copy_directory(self, source, destination):
         for filepath in source.iterdir():
             sub_destination = self.join(destination, filepath.name)
 
-            # Don't follow symlinks when uploading to storage.
-            if filepath.is_symlink():
-                log.info(
-                    "Skipping symlink upload.",
-                    path_resolved=str(filepath.resolve()),
-                )
-                continue
-
```

Comment:
I remove this check from here to allow communicating this decision, "do not follow symlinks", to the user clearly. Otherwise, with this check we are just ignoring symlinks without telling this to the user, which lead to unexpected behavior from a user perspective.